notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>This seems bad? But I know the descriptions aren't done yet - I just wasn't sure why we'd choose to do that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Great point. I was hoping to motivate that this <em>is</em> bad, and so we should therefore be careful when processing the light curve before the periodogram. I'll make this explicit. </p>